tasks/main.yml

```diff
@@ -21,9 +21,9 @@
   when: not elasticsearch_installed.stat.exists
 
 - name: Install ES plugins
-  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}remove {{ item }} && /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
-  when: head_plugin_path.stat.exists != true
+  shell: /usr/share/elasticsearch/bin/plugin {% if elasticsearch_file_name|get_major_version == 1 %}--{% endif %}install {{ item }}
   with_items: "{{ plugins_to_install }}"
+  ignore_errors: True
```

Comment:
I'll see if there's anything better in the output of that command though so we don't base it on a random string!